run_sets/testsuite.yml

```diff
@@ -55,6 +55,9 @@
 # * min_ping.feature 
 
 # **** IDEMPOTENT ******
+- features/min_bootstrap_xmlrpc.feature
+- features/min_ssh_bootstrap_xmlrpc.feature
+- features/min_bootstrap_xmlrpc_misc.feature
```

Comment:
for only 2 test i would remove the new file , and put the 2 test in bootstrap_xmlrpc.feature or/and in ssh